### PR TITLE
SRE-1118 golang builder 1.14

### PR DESCRIPTION
**Pull request by antonsuryawan**
update docker golang-builder

    - move mongodb installation in another layer
    - update script to be able to define version with default value

---

    update docker golang

    - bump up golang version to 1.14
    - bump up golangci-lint version to 1.30.0
    - update script to be able to define golang, linter and image version
    with default value

---

    update docker alpine

    - change existing Dockerfile become a template
    - update script to be able to define image version with default value

### CODE REVIEW

File: golang-builder/Dockerfile.template [hunk 1]
```diff
@@ -8,23 +8,48 @@ ENV GOCACHE="${GOPATH}/build-cache"
 RUN mkdir GOCACHE
 
 RUN echo 'http://dl-cdn.alpinelinux.org/alpine/v3.9/main' >> /etc/apk/repositories; \
-    echo 'http://dl-cdn.alpinelinux.org/alpine/v3.9/community' >> /etc/apk/repositories; \
-    apk --update add --no-cache mongodb yaml-cpp=0.6.2-r2 ca-certificates bash httpry make git curl python py-pip docker \
-    rhash openssh unzip libc6-compat openjdk8-jre gcc musl-dev python-dev libffi-dev openssl-dev libc-dev; \
-    pip install docker-compose; \
-    pip install awscli;
+	echo 'http://dl-cdn.alpinelinux.org/alpine/v3.9/community' >> /etc/apk/repositories; \
```
Comment (aripnk) [inline golang-builder/Dockerfile.template:11]: can we have a consistent character for string?, this make inconsistencies. you can choose to use ' or "

File: alpine/build.sh [hunk 2]
```diff
@@ -1,18 +1,18 @@
...
 
-readonly alpine_baseurl=http://dl-cdn.alpinelinux.org/alpine/
-readonly arch_url=${alpine_baseurl}v${1}/releases/x86_64/
-curl -OL ${arch_url}latest-releases.yaml
-readonly filename=`cat latest-releases.yaml | grep file | grep minirootfs | awk '{print $2}'`
-curl -OL ${arch_url}${filename}
+while getopts b:h option
+do
+	case "${option}"
+		in
```
Comment (aripnk) [inline alpine/build.sh:7]: looks like it is 4 spacing indentation, can we use 2?

File: golang-builder/Dockerfile.template (hunk 1 above)
Comment (antonsuryawan) [reply golang-builder/Dockerfile.template]: using double quotes now

File: alpine/build.sh (hunk 2 above)
Comment (antonsuryawan) [reply alpine/build.sh]: update editor config to use space indented with 2

**Review by aripnk — CHANGES_REQUESTED** [review]
I still found several inconsistent indentations. can we use 2 spaces for all of it?

**Pull request merged**